src/kOS/Suffixed/PartModuleField/GimbalFields.cs

```diff
@@ -20,6 +20,18 @@ private void InitializeGimbalSuffixes()
             {
                 gimbal.gimbalLock = value;
             }, "Is the Gimbal free to travel?"));
+            AddSuffix("PITCH", new SetSuffix<BooleanValue>(() => gimbal.enablePitch, value =>
+            {
+                gimbal.enablePitch = value;
+            }, "Does the Gimbal respond to pitch controls?"));
+            AddSuffix("YAW", new SetSuffix<BooleanValue>(() => gimbal.enableYaw, value =>
+            {
+                gimbal.enableYaw = value;
+            }, "Does the Gimbal respond to yaw controls?"));
+            AddSuffix("ROLL", new SetSuffix<BooleanValue>(() => gimbal.enableRoll, value =>
+            {
+                gimbal.enableRoll = value;
+            }, "Does the Gimbal respond to roll controls?"));
```

Comment:
The indentation above looks really wonky.  The braces shouldn't be at the same level as the "A" in "AddSuffix" because they're a continuation of the AddSuffix call.  They should be indented further in.